code/src/components/AlbumIcons/AlbumIcons.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './AlbumIcons.css';
+import { ReactComponent as Dots } from 'assets/icons/dots.svg';
+import { ReactComponent as Heart } from 'assets/icons/heart.svg';
+import { ReactComponent as Play } from 'assets/icons/play.svg';
+
+const AlbumIcons = (props) => {
+  return (
+    <div className="album__icons">
+      <Heart className="icon icon--smaller" />
+      <a
+        className="icon icon--interactable"
+        href={props.url}
+        target="_blank"
+        rel="noreferrer">
+        <Play className="icon--big" />
+      </a>
+      <Dots className="icon icon--smaller" />
```

Comment:
It would be nice to have a subtle hover effect on your hearts and dots and for sure the cursor: pointer indicating that they are clickable.